test/test_queue.js

```diff
@@ -2121,6 +2121,17 @@ describe('Queue', () => {
       queue.on('failed', cb);
       queue.on('error', done);
     });
+
+    it('should share child pool across all different queues created', async () => {
```

Comment:
It would be great to also test the opposite behavior, i.e. that the pools are different if we are not specifying the isSharedChildPool.